### PR TITLE
Update list view design doc to include row styling details

### PR DESCRIPTION
**Pull request by mcarrano**
## Description
This PR will update the List View design page to detail the color values for each state including hover, selected, open, edit and disabled states. This will help clarify these states to ensure consistency across repos.

## Changes

Write a list of changes this design pattern introduces

* Updated design.md

## Contribution Submission Checklist

- [ ] A GitHub issue was created and reviewed with the PatternFly community.
- [ ] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [ ] Any new files have been added to the folder for that pattern.
- [ ] The folder, for brand new patterns, has been titled with the name of the pattern.
- [ ] All images are in an "img" folder and embedded images have the correct filepath.
- [ ] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by mcarrano** [comment]
@jennyhaines I have created this PR to update the List View design page with the information on various states. If you give me the color values for each state of the row I will add them to the document. Any other comments are also welcome. I don't think mockups require any updates.

File: pattern-library/content-views/list-view/design/design.md [hunk 1]
```diff
@@ -26,7 +27,7 @@
 1. **Interaction:**
   - Hovering over a row highlights it. Single clicking anywhere in the row expands the row.
   - If the list supports both expansion and drill-down to details, a button or link must be added to the row to support the drill-down action.
-1. **Expansion Panel:** When the row is expanded, a panel is inserted between the bottom of the row and the top of the next row. All other content is pushed down the page. The panel should be sized to accommodate the content. Content can be anything that is required to convey details of a selected object. However, we do not recommend placing tables of log lists of items inside the expansion panel. This could result in nested scrolling (i.e. the master list and the detailed content require separate scroll bars).
+1. **Expansion Panel:** When the row is expanded, a panel is inserted between the bottom of the row and the top of the next row. All other content is pushed down the page. The panel should be sized to accommodate the content. Content can be anything that is required to convey details of a selected object.
```
Comment (serenamarie125) [inline pattern-library/content-views/list-view/design/design.md]: often times that "panel" has a vertical scroll bar, so it's not really sized to accommodate the content.  Should we mention something about this?

**Comment by jennyhaines** [comment]
@mcarrano - the color values are:

- **Selected/Expanded state color:**  #edf8ff (this color is custom; not in the PF palette for accessibility reasons)
- **Hover state color:**  #def3ff (pf-blue-50, and overrides the selected or expanded state color)

File: pattern-library/content-views/list-view/design/design.md (hunk 1 above)
Comment (mcarrano) [reply pattern-library/content-views/list-view/design/design.md]: I reworded this. See what you think.

**Pull request merged**